### PR TITLE
Included prepared project template

### PR DESCRIPTION
**Pull request by linhapeter**

### CODE REVIEW

File: src/style/main.scss [hunk 1]
```diff
@@ -2,3 +2,8 @@
 @use 'header.scss';
 @use 'footer.scss';
 @use 'search-section.scss';
+
+* {
```
Comment (volkov-a-dev) [inline src/style/main.scss:6]: It should be at the very beginning

File: src/template/components/footer.pug [hunk 2]
```diff
@@ -0,0 +1,27 @@
+footer.footer
+  nav.footer-nav(aria__label='footer-nav')
+    .footer-nav__div
+      h5.footer-nav__div__h5 Company
+      ul.footer-nav__div__ul
+        li.footer-nav__div__ul__li
+          a.footer-nav__div__ul__li__a(href='#') Home
```
Comment (volkov-a-dev) [inline src/template/components/footer.pug:7]: Why not write a list rendering through a loop?

File: src/template/components/navigation.pug [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+    div
+      img(src="/src/images/itapplogo_color.webp")
+    .icon-container
+      button.icon-button(href='#')
+        i.fa-regular.fa-user.fa-2xl.user-icon
+      button.icon-button.toggle-button(onclick="handleNavOpen()")
+        i.fa-solid.fa-bars.fa-2xl.menu-icon
+  .dropdown-menu.dropdown-menu_off
+    ul.dropdown-menu_on__ul
+      li.dropdown-menu_on__ul__li
```
Comment (volkov-a-dev) [inline src/template/components/navigation.pug:12]: Why not write a list rendering through a loop?

File: src/style/header.scss
Comment (volkov-a-dev) [inline src/style/header.scss:1]: Please go through the styles and get rid of unnecessary cascades in styles for example: .`navbar .icon-container .icon-button .user-icon,`

File: src/style/footer.scss
Comment (volkov-a-dev) [inline src/style/footer.scss:1]: Please go through the styles and get rid of unnecessary cascades in styles for example: .`navbar .icon-container .icon-button .user-icon,`

File: src/style/colors.scss
Comment (volkov-a-dev) [inline src/style/colors.scss:1]: Please go through the styles and get rid of unnecessary cascades in styles for example: .`navbar .icon-container .icon-button .user-icon,`

**Review by volkov-a-dev — CHANGES_REQUESTED** [review]

File: src/style/main.scss (hunk 1 above)
Comment (linhapeter) [reply src/style/main.scss:6]: * {
  box-sizing: border-box;
}

@use 'colors.scss';
@use 'header.scss';
@use 'footer.scss';
@use 'search-section.scss';   

Error: @use rules must be written before any other rules.
  ╷
5 │ @use 'colors.scss';

Comment (volkov-a-dev) [reply src/style/main.scss:6]: Why not output my style to a separate file? For example _core.scss?

**Review by volkov-a-dev — CHANGES_REQUESTED** [review]

**Pull request merged**